codalab_service.py

```diff
@@ -28,12 +27,17 @@ def print_header(description):
 
 def should_run_service(args, service):
     # `default` is generally used to bring up everything for local dev or quick testing.
-    # `default-but-worker` is generally used for real deployment since we don't
+    # `default-no-worker` is generally used for real deployment since we don't
     # want a worker running on the same machine.
     return (
         service in args.services
-        or (service != 'test' and 'default' in args.services)
-        or (service != 'test' and service != 'worker' and 'default-no-worker' in args.services)
+        or (service != 'test' and service != 'monitor' and 'default' in args.services)
```

Comment:
This logic is getting a bit convoluted...can we simplify (and generalize) by just defining a new `DEFAULT_SERVICES` list, and then making a "no" version of every service (i.e. `['no-' + service for service in ALL_SERVICES]`), so that you can do `-s default no-worker` instead?